### ISSUE TITLE
any comparison between Yake and KeyBERT?

### ISSUE THREAD

**Issue by km5ar** [LIAAD/yake#77]
any comparison between Yake and KeyBERT?

**Comment by aysebilgegunduz** [comment]
I did a subjective comparison between the two for multi-language. In my case, Yake performed better than KeyBERT according to couple of aspects. 

- KeyBert was slower than Yake
- KeyBert considers semantic meaning when suggesting keywords. In my case I wanted to summarize the text with keywords similar to each other even though they are semantically similar, yet the KeyBERT extracted the most relevant keywords that were the least similar to each other.

**Comment by km5ar** [comment]
@aysebilgegunduz  Thank you for your insights! it is super helpful!

- Did you happened to uploaded your comparison on github? just interested to see how you compared it, if not, no worries!

**Comment by aysebilgegunduz** [comment]
No, unfortunately I didn't uploaded to anywhere the dataset I used to compare was private.

**Comment by km5ar** [comment]
Oh no problem, thanks!
